app/views/layouts/_navigation.html.erb

```diff
@@ -1,39 +1,33 @@
-<nav class="navbar navbar-expand-lg bg-light fixed-top">
+<nav class="navbar navbar-expand-lg bg-light">
```

Comment:
these changes are confusing. 

may I have the source of the code where the TManagement and Plans is coming from? 

Is that something you wrote yourself as well?